config/locales/devise/en.yml

```diff
@@ -121,9 +121,13 @@ en:
       recovery_code_fallback:
         text: Don’t have access to your phone? Use a %{link} instead.
         link: recovery code
+      otp_phone_label: Phone number
+      otp_phone_label_info: Mobile or landline okay
       otp_setup: >
```

Comment:
I18n protip: we can keep this lead text in the same translation *and* bold it by taking advantage of I18n's built-in HTML support.

How:

append `_html` to the translation name (so this would be `otp_setup_html`), and add the css you need into the translation inline. Like this:

```
<strong>Every time you log in,</strong> we will send you a one-time passcode via text we will send you a one-time passcode via text message or phone call. This helps safeguard your account.
```

The Rails view will automatically interpret this string as HTML now. (don't forget to also change the key in the view to be `= t('devise.two_factor_authentication.otp_setup_html')`